hotline/notifications/views.py

```diff
@@ -29,6 +36,19 @@ def create(request):
 
 @permissions.is_active
 def list_(request):
+    # all that awesome tabs stuff
```

Comment:
We've got this same logic in two places. Can it be refactored? Maybe as a method on the user model?